### PR TITLE
Add support for go.formatTool: "goformat" #1656

### PR DESCRIPTION
**Pull request by mbenkmann**
See issue #1656

### CODE REVIEW

File: src/goInstallTools.ts [hunk 1]
```diff
@@ -33,6 +33,7 @@ const allTools: { [key: string]: string } = {
 	'gogetdoc': 'github.com/zmb3/gogetdoc',
 	'goimports': 'golang.org/x/tools/cmd/goimports',
 	'goreturns': 'github.com/sqs/goreturns',
+	'goformat': 'github.com/mbenkmann/goformat/goformat',
```
Comment (ramya-rao-a) [inline src/goInstallTools.ts]: @mbenkmann When I run `go get github.com/mbenkmann/goformat/goformat`, I get the error "package github.com/mbenkmann/goformat/goformat: cannot find package "github.com/mbenkmann/goformat/goformat" in any of:..."

Does it expect that the user has already cloned the repo?

**Comment by mbenkmann** [comment]
I verified that the installation worked from VSCode by removing the package from my GOPATH. There were no issues.
I just tried it again with a blank GOPATH and it worked.

    mkdir /tmp/go
    export GOPATH=/tmp/go
    go get github.com/mbenkmann/goformat/goformat

**Comment by mbenkmann** [comment]
I think I know what the problem is. I have symlinks in the repo to make it buildable outside of GOPATH (for people who like to download stuff in /tmp). I assume you are on Windows and those symlinks don't clone for you, right?

**Comment by mbenkmann** [comment]
Okay. I've fixed the repository. It should work without symlinks now. I've also changed the URL in goInstallTools.ts to the canonical URL which uses my own domain, because I've noticed that go get github/... creates 2 copies of the source tree, one in src/github.com/goformat and one in src/winterdrache.de/goformat

**Pull request merged**